accepted/9999-npm-audit-assertions.md

```diff
@@ -0,0 +1,70 @@
+# `npm audit` assertions
+
+## Summary
+
+A mechanism for module maintainers to assert that their modules are not impacted by advisories.
+
+```
+npm audit assert --module=<module-name> --id=<advisory identifier> --impactful=<boolean> --comment=<string with reasoning>
+```
+
+## Motivation
+
+It's relatively common that `npm audit` creates a painful experience for maintainers of transitive dependencies and for end-users, with a seemingly high likelihood to have a disproportionately large negative impact on beginners who don't actually know what's happening.
+
+Further, the amount of noise that `npm audit` generates from transitive dependencies that rely on a "vulnerable" module where there is no world in which the "vulnerability" can be exploited leads to people ignoring `npm audit` rather than leveraging it to actually address vulnerabilities.
+
+Enabling maintainers of dependencies - most importantly, dependencies that are most often transitive - to assert that their dependencies are not impacted by known vulnerabilities is a way to dramatically reduce the amount of pain that `npm audit` creates.
+
+Further, this creates an additional cascading signal of _validity_ when the maintainers of a transitive dependency mark the advisory as impactful. As maintainers make assertions, we get more context about the impact of advisories across the ecosytem, which - if surfaced to end-users through `npm audit` - helps provide more context to make an informted decision about how to address advisories that they're seeing.
+
+## Detailed Explanation
+
+- `npm audit assert` will be added with the following arguments:
+  - `--module=<module-name>` **(required)** where `<module-name>` is the name of the module having an assertion made. If I was a maintainer of `fastify` and wanted to make an assertion about it, I'd use `--module=fastify`.
```

Comment:
(nit) Why module not package? I thought NPM calls them packages everywhere.

Reply:
Indeed; a module is a file; vulns are reported against packages.